packages/tupaia-web-server/src/models/MapOverlayGroupRelation.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import {
+  MapOverlayGroupRelationModel as BaseRelationModel,
+  MapOverlayGroupRelationType as BaseRelationType,
+} from '@tupaia/database';
+import { Model } from '@tupaia/server-boilerplate';
+
+type MapOverlayGroupRelationFields = Readonly<{
+  id: string;
+  map_overlay_group_id: string;
+  child_id: string;
+  child_type: 'mapOverlay' | 'mapOverlayGroup';
+  sort_order: number | null;
+}>;
```

Comment:
I think we can get this from the `@tupaia/types` package now. 
```suggestion
import { MapOverlayGroupRelation } from '@tupaia/types';
```